### PR TITLE
Don't use a template for the output imagestream

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

/assign @csrwng 
depends on https://github.com/openshift/release-library/pull/26

### CODE REVIEW

**Comment by csrwng** [comment]
actually cancelling my lgtm :)

File: cluster/ci/origin/pipelines/build/Jenkinsfile [hunk 1]
```diff
@@ -40,13 +40,8 @@ pipeline {
...
     }
     stage ("Create ImageStream") {
-      when {
-        not { expression { return imageStreamExists(this, "${this._buildName}") } }
-      }
       steps {
-        echo "Creating image stream ${this._buildName}"
-        applyTemplate(this, "${templateBase}/output-imagestream.yaml",
-          "BUILD_NAME=${this._buildName}")
+        ensureImageStream("${this._buildName}")
```
Comment (csrwng) [inline cluster/ci/origin/pipelines/build/Jenkinsfile]: pass 'this' as the first argument

File: cluster/ci/origin/pipelines/test-build/Jenkinsfile [hunk 2]
```diff
@@ -41,13 +41,8 @@ pipeline {
...
     }
     stage ("Create ImageStream") {
-      when {
-        not { expression { return imageStreamExists(this, "${this._buildName}") } }
-      }
       steps {
-        echo "Creating image stream ${this._buildName}"
-        applyTemplate(this, "${templateBase}/output-imagestream.yaml", 
-          "BUILD_NAME=${this._buildName}")
+        ensureImageStream("${this._buildName}")
```
Comment (csrwng) [inline cluster/ci/origin/pipelines/test-build/Jenkinsfile]: pass 'this' as the first argument

**Comment by csrwng** [comment]
minor change requested, otherwise lgtm

**Comment by stevekuznetsov** [comment]
Fixed

**Pull request merged**